src/components/fieldText/fieldText.tsx

```diff
@@ -132,10 +135,9 @@ export const FieldText = forwardRef<HTMLInputElement, FieldTextProps>(
     return (
       <>
         {label && (
-          <span className={cx('label', { disabled })}>
+          <FieldLabel isRequired htmlFor={inputId}>
```

Comment:
It should not be required by default
```suggestion
          <FieldLabel isRequired={isRequired} htmlFor={inputId}>
```

Reply:
done